dipti2jcommerce/magento2
# [Backport] fixed issue #22736 - Cursor position not in right side of search keyword in mobile

### Original Pull Request 
 https://github.com/magento/magento2/pull/22795
### Description (*)
For mobile view when search is focused I have set with only found solution https://www.google.com/search?q=jquery+set+cursor+at+end+of+input

### Fixed Issues (if relevant)
1. magento/magento2#22736: 
Cursor position not in right side of search keyword in search box when click on search again (Mobile issue)

### Manual testing scenarios (*)
1. Load website in mobile view
2. Search anything.
3. Click on search icon to focus search box
4. Check the cursor position

### Contribution checklist (*)
